content/enterprise_influxdb/v1.8/administration/config-data-nodes.md

```diff
@@ -1222,32 +1229,21 @@ ciphers = [ "TLS_ECDHE_ECDSA_WITH_CHACHA20_POLY1305",
             "TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384"
 ]
 
-min-version = "tls1.2"
+min-version = "tls1.3"
 
-max-version = "tls1.2"
+max-version = "tls1.3"
 
 ```
 
-#### `min-version = "tls1.2"`
+#### `min-version = "tls1.3"`
```

Comment:
@pierwill, please verify these are same TLS updates you made in 1.9.

Reply:
Looks good!